test_vault/.obsidian/plugins/obsidian-meals/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "tmayoff-meals",
     "name": "Meal Plan",
-    "version": "0.0.8",
+    "version": "0.0.9",
```

Comment:
undo this, I don't want to make a release with this PR immediatly

Reply:
That’s the current version, surely? Not a change I made anyway?

Reply:
oh this is in the test vault, I think this file was updated when you started the extension locally, I made the version bump in github online so it didn't update the this file. 

You can ignore this then